test/cli_copy_resource_test.go

```diff
@@ -0,0 +1,65 @@
+package test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/fnproject/cli/testharness"
+)
+
+func TestResourceCopy(t *testing.T) {
+
+	const testFuncSource = `package main
```

Comment:
Can we move the function and yaml sources to their own directory under `test` rather than including them inline? See for example https://github.com/fnproject/cli/tree/206ff749b2f2251e90e89f37bba5d4739d82ed56/test/simplefunc